src/main/java/com/sapozhnykov/dao/cart/CartDao.java

```diff
@@ -0,0 +1,34 @@
+package com.sapozhnykov.dao.cart;
+
+import com.sapozhnykov.domain.Product;
+
+import java.util.ArrayList;
+
+public interface CartDao {
```

Comment:
It's looks like ProductDao

Reply:
what about update?

Reply:
At the last lesson, Artem said that we would not have a Cart. I removed it from the project.